docs/server/settings.rst

```diff
@@ -294,20 +298,50 @@ Translation environment configuration settings.
             'HOST': 'localhost',
```

Comment:
Why not change this entry to ``local`` instead of ``default``, since default doesn't mean anything anymore